pyseir/cli.py

```diff
@@ -59,6 +61,12 @@ def entry_point():
     """Basic entrypoint for cortex subcommands"""
     dataset_cache.set_pickle_cache_tempdir()
     sentry_sdk.init(os.getenv("SENTRY_DSN"))
+    structlog.configure(
+        processors=[
+            structlog.stdlib.add_log_level,  # required before SentryProcessor()
+            SentryProcessor(level=logging.INFO),
```

Comment:
Info feels like it might be a bit verbose?

Reply:
I don't think it creates an event for every message passed to the sentry_sdk. Added comment `sentry_sdk creates events for level >= ERROR and keeps level >= INFO as breadcrumbs`